CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ option (NLOPT_PYTHON "build python bindings" ON)
 option (NLOPT_OCTAVE "build octave bindings" ON)
 option (NLOPT_MATLAB "build matlab bindings" OFF)
 option (NLOPT_GUILE "build guile bindings" ON)
+option (NLOPT_JAVA "build java bindings" ON)
```

Comment:
a new documentation section will be needed

Reply:
I will look into it.

Reply:
The latest revision (commit fe30d98be818f511b5e6759dde8bc56297c1da3a) adds the missing documentation.